pkg/query-service/constants/constants.go

```diff
@@ -30,7 +30,10 @@ func GetAlertManagerApiPrefix() string {
 	return "http://alertmanager:9093/api/"
 }
 
-const RELATIONAL_DATASOURCE_PATH = "/var/lib/signoz/signoz.db"
+// Alert manager channel subpath 
+var AmChannelApiPath = GetOrDefaultEnv("ALERTMANAGER_API_CHANNEL_PATH", "v1/routes")
+
+var RELATIONAL_DATASOURCE_PATH = GetOrDefaultEnv("SIGNOZ_LOCAL_DB_PATH", "/var/lib/signoz/signoz.db")
```

Comment:
`SIGNOZ_LOCAL_DB_PATH` might not be a correct name as it might be pointing to `postgres` instance later. We should use something more generic.

Reply:
When postgres arrives, we can introduce SIGNOZ_DB* parameters such as SIGNOZ_DB_TYPE (postgres/local), SIGNOZ_DB_USER_NAME, SIGNOZ_DB_PASSWORD, SIGNOZ_DB_HOST, SIGNOZ_DB_PORT in addition to SI*_DB_PATH which will work for sql*lite 

I can change the name to SIGNOZ_SQLLITE_DATA_PATH to indicate it only applies to sql*lite.